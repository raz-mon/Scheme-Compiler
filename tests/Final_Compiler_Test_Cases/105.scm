(define world "world")
(define string-append (lambda (x y) "hello world"))
(define format (lambda (x y) y))
"(~{hello {world}})"
