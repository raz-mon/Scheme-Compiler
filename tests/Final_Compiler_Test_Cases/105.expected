(define world "world")
(string-append "hello " (format "~a" world))